src/stories/theme/_horizontal-line.css

```diff
@@ -1,3 +1,5 @@
 .ck-content hr {
-  margin: var(--ck-spacing-unit) 0;
+  margin: var(--ck-spacing-extra-large) 0;
```

Comment:
There might need to be a notion of padding if `border-radius` focus is to be applied to this element.

Reply:
I've updated the horizontal line spacing to match the expected focus-ring size. I'll update any required padding/margin when we handle all of the content spacing in a following PR.